base32_identifier/base32.py

```diff
@@ -81,6 +80,31 @@ def encode(number, split_every=0, min_length=0, checksum=False):
     return encoded
 
 
+def generate(length=8, split_every=0, checksum=False):
+    """Generate random base32 identifier.
```

Comment:
Minor: string instead of identifier, since now we are renaming the repo?

Reply:
done